src/cc/compat/linux/types.h

```diff
@@ -53,7 +53,7 @@ typedef int bool;
 #define ENOSPC 28
 #define PAGE_SIZE 4096
 #define PAGE_MASK (~(PAGE_SIZE-1))
-#define THREAD_SIZE 8192
+#define THREAD_SIZE 16384
```

Comment:
I suggest you use the full definition to make it clearer, i.e.
```c
#define THREAD_SIZE  (PAGE_SIZE << THREAD_SIZE_ORDER)
```

Reply:
K will add